lib/sql.js

```diff
@@ -426,12 +426,7 @@ SQLConnector.prototype.propertyName = function(model, column) {
  * @returns {String} The id column name
  */
 SQLConnector.prototype.idColumn = function(model) {
-  var name = this.getDataSource(model).idColumnName(model);
-  var dbName = this.dbName;
-  if (typeof dbName === 'function') {
-    name = dbName(name);
-  }
-  return name;
+  return this.getDataSource(model).idColumnName(model);
```

Comment:
This change looks really suspicious to me, you are effectively removing support for `this.dbName` allowing connectors to provide their own code for dealing with FK column names.

I'd like us all to better understand why is `dbName` provided by the postgresql connector causing problems when called in LB4 context, especially considering that it works fine in LB3.

I strongly suspect we are fixing the problem in a wrong place.